frontend/packages/dev-console/integration-tests/features/monitoring/dashboard-display.feature

```diff
@@ -0,0 +1,98 @@
+        And user will see the Memory Usage
+        And user will see Receive Bandwidth
+        And user will see Transmit Bandwidth
+        And user will see the Rate of Received Packets
+        And user will see the Rate of Transmitted Packets
+        And user will see the Rate of Received Packets Dropped
+        And user will see the Rate of Transmitted Packets Dropped
+
+
+    @regression
+    Scenario: Dashboard tab on the Monitoring page for all workloads: MH-01-TC02
+        Given user has created workload "national-parks-test-1" with resource type "Deployment"
+        And user opened the url of the workload "national-parks-test-1" in topology page
+        And user is on Monitoring page
+        When user clicks on "Dashboard" tab
+        Then user will see the dropdown selected with All Workloads by default
+        And user will see the CPU Usage
+        And user will see the Memory Usage
+        And user will see Receive Bandwidth
+        And user will see Transmit Bandwidth
+        And user will see the Rate of Received Packets
+        And user will see the Rate of Transmitted Packets
+        And user will see the Rate of Received Packets Dropped
+        And user will see the Rate of Transmitted Packets Dropped
+
+
+    @smoke
+    Scenario: Events tab on the Monitoring Page: MH-01-TC05
+        Given user is on Monitoring page
+        When user clicks on "Events" tab
+        Then user will see events related to all resources and all types
+
+
+    @regression
+    Scenario: Events for Multiple Resources: MH-01-TC06
+        Given user has workloads of all resource types
+        And user is on Monitoring page
+        When user clicks on "Events" tab
+        And user clicks on Resources dropdown
+        And user selects Service
+        And user selects Deployment
+        And user selects DeploymentConfig
+        Then user will see events for Service, Deployment and DeploymentConfig type resources
+
+
+    @regression
+    Scenario Outline: Event types on monitoring page: MH-01-TC07
+        Given user has workloads of all resource types
+        And user is on Monitoring page
+        When user clicks on "Events" tab
+        And user clicks on Types dropdown
+        And user selects "Normal" from Types dropdown
+        Then user will see normal types of events
+
+        Examples:
+        | Type    |
+        | Normal  |
+        | Warning |
+
+
+    @regression
+    Scenario: Filter Events by name or message: MH-01-TC08
+        Given user has workloads of all resource types
+        And user is on Monitoring page
+        When user clicks on "Events" tab
+        And user enters "Scaled Up" in the Filter field
+        Then user will see events having Scaled Up message
+
+
+    @smoke
+    Scenario: Alerts details on monitoring page: MH-01-TC04
+        Given user is on Monitoring page
+        When user clicks on "Alerts" tab
+        Then user is able to see Name, Severity, Alert State and Notifications
```

Comment:
This may not work if we have some alerting rules configured on namespace other wise it'll show `no Alerts found` .

Reply:
How do we know whether alerts are configured or not, @invincibleJai - Do we have any document for the same
@gajanan-more - Any idea about this alerts part?

Reply:
So for that particular scenario, we need to add the alerts for that namespace. If there are no alerts then we won't be able see Alerts section on the sidebar
8:33
We need to apply few YAMLs first to get some alerts in the namespace then it will show alerts section on the sidebar
8:33
I have few YAMLs but I am not sure about whether they will work or not
8:34
These alerts are firing alerts and related to Version
8:34
Here are the YAMLs:
To test monitoring alerts:
oc apply -f https://gist.githubusercontent.com/vikram-raj/d801e9ef6400cc1bd5af5a336bbedfa0/raw/61d90b705617ba8070da80f8c97ac16376a9ca58/prometheus-example-app.yaml
oc apply -f https://gist.githubusercontent.com/vikram-raj/86300d923a7a3a3a3348e359327e2307/raw/535d889923141a3f3e26e355d6bccec4650a0fcd/user-workload-config.yaml

Reply:
@gajanan-more @makambalaji There is a change in how we setup the alerting rule in the cluster. We need to apply this yaml now. https://gist.githubusercontent.com/vikram-raj/c14c080f33c5bdd4125f3ba9e9096509/raw/6ae3bd8cb2aff873aad76b6edb9d893117d0bf55/prometheus-example-app.yaml

Reply:
Included as manual step for now